release-notes/opensearch-dashboards-reporting.release-notes-2.11.1.0.md

```diff
@@ -0,0 +1,12 @@
+## Version 2.11.1 Release Notes
+
+Compatible with OpenSearch and OpenSearch Dashboards Version 2.11.0
+
+### Features
```

Comment:
@pjfitzgibbons can you cross check/ this looks like a maintainence or bug fix / not a feature

Reply:
Yes, correct, #229 fixes #227.  Not a feature.